[BUG] Fix importlib resources/metadata on older Python versions

melroy89 opened the pull request:
Trying to fix #153...

- [x] Add `importlib-metadata` & `importlib-resources` as dependencies
- [x] Check on specific Python version regarding the importlib resources (python v3.10 or higher) in order to have `files()` working
- [x] `importlib_metadata` should now also work on Python version before 3.8
- [x] Remove obsolete `MANIFEST.in` file

melroy89 committed:
Invert the the imports

jayvdb commented:
If you do this, you need to specify a minimum version of PyPI package `importlib_resources` as a dependency

melroy89 committed:
Before Python 3.10 fallback to importlib-resources package

melroy89 commented:
I changed it a bit more. I added an explicit check for Python 3.10 and higher: https://github.com/fake-useragent/fake-useragent/pull/154/files#diff-eaceb65d44c0eb2ed1b5108e3e0863edd4c31e7d16d3f7ac5e041ae70b87ce8bR13

Also I added the dependency only for Python lower than 3.10: 

https://github.com/fake-useragent/fake-useragent/pull/154/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R13

I also noticed that the unit tests are running much faster now. Most likely because it's actually using the local data as it should be across all Python versions 👍🏽 .

melroy89 committed:
Increase patch version

melroy89 committed:
Add also importlib metadata as dependency

melroy89 committed:
Add missing validate pyproject back

melroy89 committed:
Remove MANIFEST.in. Also only list additional test deps in the tox file
